src/Path.Drag.js

```diff
@@ -18,7 +18,7 @@ L.PathDraggable = L.Draggable.extend({
 
   _onDown: function (e) {
     var first = e.touches ? e.touches[0] : e;
-    this._startPoint = new L.Point(first.clientX, first.clientY);
+    this._startPoint = this._path._map.mouseEventToLayerPoint(first);
     if (this._canvas && !this._path._containsPoint(this._startPoint)) { return; }
```

Comment:
`!this._path._containsPoint(this._startPoint)` is true, which means the point is not inside the polygon. The `_pxBounds` of polygon is `max(x:238.5y:168.5),min(x:119.5y:10.5)` while the `_startPoint` is `x:272,y:118`.